tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py26,py25,py32
+envlist = py27,py33,py34,py35,py36
```

Comment:
This change is great, but can you make it a separate commit?

Reply:
Will redo the commits tomorrow from work...